schemas/constructs/v1alpha1/capability.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$id": "https://schemas.meshery.io/capability.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Meshery manages entities in accordance with their specific capabilities. This field explicitly identifies those capabilities largely by what actions a given component supports; e.g. metric-scrape, sub-interface, and so on. This field is extensible. Entities may define a broad array of capabilities, which are in-turn dynamically interpretted by Meshery for full lifecycle management.",
+  "additionalProperties": false,
+  "type": "object",
+
+  "required": ["schemaVersion", "version", "displayName", "kind", "type", "state", "status"],
```

Comment:
kind, type and subtype together should uniquely identify a capability. Hence subType should be required

Reply:
@MUzairS15, my guess is the thinking here is that it's not always necessary to have this level specificity in order to identify a capability. Given this, that begs the idea that `subtype` is optional.

If we can put some forethought into a proposed set of classifications (a set of examples of how kind, type, subtype can be made to always be present), that exercise will lend itself to making it comfortable to require subtype.